### DISCUSSION TITLE
Disable Collision for Player

### DISCUSSION THREAD

**Discussion by cemalgnlts** [excaliburjs/Excalibur#2728] [Q&A] [answered]
Hi,

I have a bomb object in the game. When the player drops the bomb, he cannot interact with it, but enemies can push (throw) the bomb.

So, bomb's physics (body) features must be disabled for the player.
*(The bomb must ignore player)*

Can i do this?

**Comment by cemalgnlts** [comment]
I discovered CollisionGroups but I'm not sure how to add the floor to a group since I'm using tiledMap.

**Reply by eonarheim** [reply, marked as answer]
Hi @cemalgnlts! Great question!

I'd definitely use a collision group to make this happen. If the player and the bomb are in the same collision group they won't interact.

There is some additional documentation on collision groups here: https://excaliburjs.com/docs/api/edge/classes/CollisionGroup.html

```typescript
const playerGroup = ex.CollisionGroupManager.create('player');
const player = new ex.Actor({
  ...
  collisionGroup: playerGroup
});

// drop bomb code
const bomb = new ex.Actor({
  ...
  collisionGroup: playerGroup
});

```

You can add a collision group to a tile map by retrieving it's body component and setting the group property. Apologies TileMap's don't have the same convenient way as Actors to set the collision group. By default body components have the special group `ex.CollisionGroup.All` that collides with everything.

```typescript
const floorGroup = ex.CollisionGroupManager.create('floor');

const tilemap = new ex.TileMap({...});
const body = tilemap.get(ex.BodyComponent);
body.group = floorGroup;
```

**Reply by cemalgnlts** [reply]
Yes, it's reacting as I want it now, thanks.
